ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml

```diff
@@ -954,6 +976,7 @@ SFSS:
   - supply_fan_run_time_accumulator
   - failed_supply_fan_alarm
   - supply_fan_lost_power_alarm
+  - local_override_alarm
```

Comment:
TBD on if this is specific enough (as there are other components that could be locally overridden) - chat w Trevor